perun-core/src/main/java/cz/metacentrum/perun/core/blImpl/GroupsManagerBlImpl.java

```diff
@@ -3339,6 +3339,7 @@ private void categorizeMembersForLightweightSynchronization(PerunSession sess, G
 							Candidate candidate = new Candidate(user, source);
 							//for lightweight synchronization we want to skip all update of attributes
 							candidate.setAttributes(new HashMap<>());
+							candidate.setExpectedStatus(subjectFromLoginSource.get("status"));
```

Comment:
This should work only for users that were not yet in the group - don't we want to update status for existing members as well? (Even during lightweight synchronization). Now it only validates the member, but ideally we want to check the status in the ExtSource and set accordingly.